api/models/model_endpoint_alert.go

```diff
@@ -152,7 +152,8 @@ func (alert ModelEndpointAlert) ToPromAlertSpec(dashboardBaseURL string) (PromAl
 	// Lint alert to comply PromQL
 	for i, group := range spec.Groups {
 		for j, rule := range group.Rules {
-			exp, err := promql.ParseExpr(rule.Expr.Value)
+			parser := parser.NewParser(rule.Expr.Value)
+			exp, err := parser.ParseExpr()
```

Comment:
`promql.ParseExpr` is now deprecated (completely gone). To do the same thing we need to first [create](https://github.com/prometheus/prometheus/blob/main/promql/parser/parse.go#L42) a `parser` object.